### DISCUSSION TITLE
Screen capture videos - drag and drop

### DISCUSSION THREAD

**Discussion by wendellpiez** [usnistgov/oscal-xproc3#18] [Show and tell]
Having uncovered a means of producing a screen capture video (in hope and with trepidation), presenting --

*NB* - YMMV! Tested on Windows 10. If you have advice or expertise to offer on how to do this on other platforms, let us know. (With apologies to Gates, Jobs and Xerox PARC.)

## Dragging and dropping a pipeline (power-up smoke test)

https://github.com/user-attachments/assets/0ae03fcd-efb8-4d16-a829-c773fb3f18a7

## Dragging and dropping a pipeline (schema field test example)

https://github.com/user-attachments/assets/3e57d214-88bf-4568-b880-b5bb192d4110

## Setting up XProc for drag and drop under Windows

https://github.com/user-attachments/assets/3aff78b6-41f8-4cf5-9b16-4844d6989fa5

**Comment by iMichaela** [comment]
@wendellpiez - Mac OS has native screen capture application. It can be launched with Shift-Command-5.
Here it is the demo.

https://github.com/user-attachments/assets/46045700-27e4-486c-9336-f0437f3dea22

**Reply by wendellpiez** [reply]
Thanks Michaela -- but what I don't know is how to set up the drag and drop functionality - dropping the XProc file onto the script (or shortcut), icon selection etc. -- Applescript (I date myself)? What works on a Mac or Linux desktop environment these days?
